src/VisualStudio/Core/Impl/Options/Style/BooleanCodeStyleOptionViewModel.cs

```diff
@@ -58,7 +58,7 @@ public override NotificationOptionViewModel SelectedNotificationPreference
             string groupName,
             List<CodeStylePreference> preferences = null,
             List<NotificationOptionViewModel> notificationPreferences = null)
-            : base(option, description, truePreview, falsePreview, info, options, groupName, preferences, notificationPreferences)
+            : base(option, description, new[] { truePreview, falsePreview }, info, options, groupName, preferences, notificationPreferences)
```

Comment:
So are we passing data down to the base class but the base class can't do anything without calling GetPreview? Why not just hold the data at this class's level?